packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingToolMarks/DrawingToolMarks.js

```diff
@@ -57,6 +57,14 @@ function DrawingToolMarks ({ activeMarkId, marks, onDelete, onDeselectMark, onFi
       }
     }
 
+    function endMoveMark (event) {
```

Comment:
This seems to be a duplicate of lines 52-58. Rebase mistake?

Reply:
It was indeed a rebase mistake, removed.